### PR TITLE
Documentation for the FiveM integration

### PR DESCRIPTION
**Pull request by Sander0542**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/65089
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3116
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Sander0542** [comment]
The `ha_release` property needs to be updated. Should this be 2022.3?

**Comment by thecode** [comment]
> The `ha_release` property needs to be updated. Should this be 2022.3?

Yes, thanks

File: source/_integrations/fivem.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ha_config_flow: true
+ha_codeowners:
+  - '@Sander0542'
+ha_domain: fivem
+ha_platforms:
+  - binary_sensor
+  - sensor
+---
+
+FiveM allow players to play the game [Grand Theft Auto V](https://www.rockstargames.com/V) by [Rockstar Games](https://www.rockstargames.com) online with other players. FiveM adds support for custom resources. The `FiveM` integration lets you retrieve information from a FiveM server within Home Assistant.
```
Comment (frenck) [inline source/_integrations/fivem.markdown]:
```suggestion
FiveM allows players to play the game [Grand Theft Auto V](https://www.rockstargames.com/V) by [Rockstar Games](https://www.rockstargames.com) online with other players. FiveM adds support for custom resources. The FiveM integration lets you retrieve information from a FiveM server within Home Assistant.
```

**Review by frenck — APPROVED** [review]
Thanks, @Sander0542 👍

**Pull request merged**